WalletWasabi/Helpers/EnvironmentHelpers.cs

```diff
@@ -159,12 +160,17 @@ public static string ExtractFileName(string callerFilePath)
 			return fileNameWithoutExtension;
 		}
 
+		public static void ShellExec(string cmd, bool waitForExit = true)
+		{
+			ShellExecAsync(cmd, waitForExit).GetAwaiter().GetResult();
```

Comment:
Remove this function.

Reply:
I removed this function and replace it everywhere by the async version but then you requested to undo the changes inthe `TorProcessManager` and for that reason I had to create this sync version. In this way the `TorProcessManager` can be kept unmodified.

Reply:
Done.